apps/webstore/src/app/pages/auth/checkout/checkout.component.ts

```diff
@@ -236,13 +249,81 @@ export class CheckoutComponent implements OnInit {
   }
 
   submitCheckout() {
+    // check checkout form and if terms are accepted
+    if (!this.checkoutForm.valid || !this.isTermsAndConditionsAccepted) {
```

Comment:
When this if statement is run, it doesn't display if there are issues in the billing address. 
Thats why it would require the same code seen on 260-267 inside this if statement. 

Could be reworked, but this is min requirement.

Reply:
You could also add all checks into one function instead of having check checkout form and one for billing.

checkForms(){
// check checkoutForm with forloop

    if( !isBillingAddressTheSame ) {
        // check billingForm with forloop
    } 
}

Reply:
You mean one function with all the if logic from line 253-267 or do you mean the input fields?

Reply:
Added a function for the logic.